### PR TITLE
feat(TCOMP-2452) api test coverage for healthcheck action

### PR DESCRIPTION
**Pull request by lxia-talend**
### Requirements
https://jira.talendforge.org/browse/TCOMP-2452

### Why this PR is needed?
Add coverage

### What does this PR adds (design/code thoughts)?
New test and improve the sample connector

### CODE REVIEW

File: sample-parent/sample-connector/src/main/java/org/talend/sdk/component/test/connectors/service/UIService.java [hunk 1]
```diff
@@ -115,4 +124,36 @@ public ValidationResult retrieveValidation() throws Exception {
...
+     * Type: healthcheck
+     * API: @org.talend.sdk.component.api.service.healthcheck.HealthCheck
+     *
+     * Returned type: org.talend.sdk.component.api.service.healthcheck.HealthCheckStatus
+     */
+    @HealthCheck(HEALTH_CHECK_OK)
+    public HealthCheckStatus discoverHealthcheckOk(@Option final TheDatastore unused) {
+        try {
+            return new HealthCheckStatus(Status.OK, "HealthCheck Success");
+        } catch (Exception e) {
```
Comment (acatoire) [inline sample-parent/sample-connector/src/main/java/org/talend/sdk/component/test/connectors/service/UIService.java:140]: I don't think that you still need the try/catch

File: sample-parent/sample-connector/src/main/resources/org/talend/sdk/component/test/connectors/Messages_en.properties [hunk 2]
```diff
@@ -24,3 +24,5 @@ the_family.dataset.TheDataset._displayName = Name: Sample dataset -en
 the_family.actions.suggestions.action_LIST_ENTITIES._displayName = Name: LIST ENTITIES -en
 the_family.actions.update.action_UPDATE._displayName = Name: Update the nested config elements -en
 the_family.actions.validation.action_VALIDATION._displayName = Name: Validate the configuration -en
+the_family.actions.healthcheck.action_HEALTH_CHECK_OK._displayName = Name: Check the HealthCheck status OK -en
+the_family.actions.healthcheck.action_HEALTH_CHECK_KO._displayName = Name: Check the HealthCheck status KO -en
```
Comment (acatoire) [inline sample-parent/sample-connector/src/main/resources/org/talend/sdk/component/test/connectors/Messages_en.properties:28]: Can you check if the result is the same with python script?

**Review by acatoire — CHANGES_REQUESTED** [review]
Some minor change requested after code review.

File: sample-parent/sample-connector/src/main/java/org/talend/sdk/component/test/connectors/service/UIService.java (hunk 1 above)
Comment (lxia-talend) [reply sample-parent/sample-connector/src/main/java/org/talend/sdk/component/test/connectors/service/UIService.java]: OK, remove the try/catch

File: sample-parent/sample-connector/src/main/resources/org/talend/sdk/component/test/connectors/Messages_en.properties (hunk 2 above)
Comment (lxia-talend) [reply sample-parent/sample-connector/src/main/resources/org/talend/sdk/component/test/connectors/Messages_en.properties]: The only difference it that missing Line break symbol.  Already use the python script file to update the messages properties.

**Review by acatoire — APPROVED** [review]
Good and ready to merge

**Pull request merged**